### ISSUE TITLE
Exception thrown when I open PS: TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult'

### ISSUE THREAD

**Issue by carlossanlop** [PowerShell/PSReadLine#2771]
## Environment

PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0-preview.8
OS: Microsoft Windows 10.0.19043
BufferWidth: 296
BufferHeight: 44

## Exception report

Last 0 Keys
```

```
None (This happens when I open PS).

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x60004a6+0x0
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset() in Microsoft.PowerShell.PSReadLine2.dll:token 0x6000478+0x7
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000f2+0x118
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken) in Microsoft.PowerShell.PSReadLine2.dll:token 0x60000eb+0x60
```

## Steps to reproduce

Open PowerShell 7 Preview. The exception shows up without any input.

Press enter. The exception shows up again.

Press tab. The exception shows up again.

Note: This happened after installing PowerShell Preview from the Microsoft Store today.

## Expected behavior

No error.

## Actual behavior

Exception on almost any input.

Note: I can still "use" powershell and execute commands, as long as I don't press tab. But once I press enter, the exceptionb shows up again (while the command executes in parallel).

**Comment by iSazonov** [comment]
You should use latest PSReadLine v2.2.0-beta3 since there is a breaking change in prediction interface API.

**Comment by dsghi** [comment]
Beta 3 does resolve this error.

**Comment by daxian-dbw** [comment]
Yeah, this is a known issue and has been addressed in 2.2.0-beta3 release of PSReadLine. Please upgrade to use 2.2.0-beta3 with the latest 7.2.0-preview.9 PowerShell.
